src/design-system/profile-dropdown/profile-dropdown-wallet-status.css.ts

```diff
@@ -9,6 +9,8 @@ export const button = style([
   }),
 ]);
 
+export const flat = style([]);
```

Comment:
p.s. maybe we don't need this here but just wanted to point it out in case additional styles are necessary